src/algorithms/tracking/TrackProjector.cc

```diff
@@ -153,7 +153,7 @@ namespace eicrecon {
                         static_cast<float>(boundCov(Acts::eBoundPhi, Acts::eBoundQOverP))
                 };
                 const float time{static_cast<float>(boundParams(Acts::eBoundTime))};
-                const float timeError{sqrt(static_cast<float>(boundCov(Acts::eBoundTime, Acts::eBoundTime)))};
+                const float timeError{static_cast<float>(sqrt(static_cast<float>(boundCov(Acts::eBoundTime, Acts::eBoundTime))))};
```

Comment:
I don't get why return type does not match, but okay. We might as well let sqrt to be calculated on the original type.
```suggestion
                const float timeError{static_cast<float>(sqrt(boundCov(Acts::eBoundTime, Acts::eBoundTime)))};
```

Reply:
Good suggestion, but I'm not yet sure why this suddenly came up.